react/src/lib/components/DeckGLMap/DeckGLMap.tsx

```diff
@@ -81,6 +86,7 @@ const DeckGLMap: React.FC<DeckGLMapProps> = ({
     onMouseEvent,
     selection,
     getTooltip,
+    distanceRulerStyle,
```

Comment:
I don't recommend adding a new prop to the Map component. There is a scale prop for distanceScale component, you can update that prop to pass styles.